chandra_aca/drift.py

```diff
@@ -9,45 +9,39 @@
 https://github.com/sot/aimpoint_mon/blob/master/fit_aimpoint_drift-2018-11.ipynb
 """
 import functools
+import json
 import os
 from pathlib import Path
 
 import numpy as np
 from astropy.table import Table
 from astropy.utils.data import download_file
 from Chandra.Time import DateTime
+from ska_helpers.utils import LazyDict
+
+
+# Define path to best fit model parameters for ACA drift model.
+# See notebooks fit_aimpoint_drift_*.ipynb for fit details.
+def DRIFT_MODEL_PATH():
```

Comment:
I wasn't sure if you'd want something like get_xija_model_spec but this seems fine and makes sense as being specific to this drift model.